apps/condo/domains/meter/access/MeterReading.js

```diff
@@ -76,6 +76,19 @@ async function canManageMeterReadings (args) {
         return await checkPermissionsInEmployedOrRelatedOrganizations(context, user, meterOrganization, 'canManageMeterReadings')
     }
 
+    if (operation === 'update' && itemId) {
+        const meterReading = await getByCondition('MeterReading', {
+            id: itemId,
+            deletedAt: null,
+        })
+        if (!meterReading) return false
+
+        const meterReadingOrganization = get(meterReading, 'organization')
+        if (!meterReadingOrganization) return false
```

Comment:
We also checks that organization is not undefined inside the checkPermissionsInEmployedOrRelatedOrganizations, but it’s ok to check it here too.